### DISCUSSION TITLE
Field: Time Interface - Jump with Tabs

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#16757] [Feature Requests]
Curently it is very frustrating to create a businesshour element. More frustrating is, that the time field is not very user friendly, since i cant jump to the next time using tab. Also clicking inside of it is very annoying.

Edit:
It would be also nice the be able to type in the time directly.

<img width="809" alt="Bildschirmfoto 2022-12-08 um 15 11 41" src="https://user-images.githubusercontent.com/33295329/206467771-8c343852-bf05-41af-a945-a0f4548b9103.png">

**Comment by NilsBaumgartner1994** [comment]
Please allow to select a different interface.

**Reply by NilsBaumgartner1994** [reply]
A current workaround is to use a "Input" Field: with a Validation Regex: `^(2[0-3]|[01]?[0-9]):([0-5]?[0-9])$`

<img width="604" alt="Bildschirmfoto 2022-12-08 um 15 29 23" src="https://user-images.githubusercontent.com/33295329/206472282-0296e62e-a59d-48fb-b53e-469cc8ee016f.png">

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
